t/unit/utils/test_platforms.py

```diff
@@ -1001,3 +1009,11 @@ def test_check_privileges_with_c_force_root_and_no_group_entry(accept_content, r
             check_privileges(accept_content)
 
         assert recwarn[0].message.args[0] == ASSUMING_ROOT
+
+
+def test_skip_checking_privileges_when_grp_is_unavailable(recwarn):
+    with patch('celery.platforms.try_import',
+               return_value=None):
```

Comment:
Probably `patch("celery.platforms.grp", new=None)` and just rely on magic knowledge of the `grp` name - I can't think of a practical alternative that wouldn't cause more potential confusion due to reloading the module and the heavy use of from imports in the celery codebase.